src/data/finsupp/basic.lean

```diff
@@ -699,6 +699,13 @@ lemma on_finset_prod {s : finset α} {f : α → M} {g : α → M → N}
   (on_finset s f hf).prod g = ∏ a in s, g a (f a) :=
 finset.prod_subset support_on_finset_subset $ by simp [*] { contextual := tt }
 
+@[to_additive] lemma prod_eq_prod_fintype {N β : Type*} [comm_monoid N] [fintype α]
```

Comment:
Some other name ideas:

* `prod_eq_prod_univ`
* `prod_eq_fintype_prod` (since we have `fintype.prod_*` lemmas elsewhere)

I slightly prefer the univ spelling, let's see if any other reviewers agree before changing what you have

Reply:
OK, I'm fine with both, personally. I will need to change them in LTE, at any rate.

Reply:
@eric-wieser What would you like me to do to accept this `PR`? Should I invite some other reviewer?

Reply:
Yes, please do!

Reply:
Ok, I will.